numba_dpex/experimental/launcher.py

```diff
@@ -145,9 +179,47 @@ def populate_kernel_args_and_argsty_arrays(
             array_of_kernel_arg_types=args_ty_list,
         )
 
+    def populate_meminfo_arrays(
+        self,
+        kernel_argtys: tuple[types.Type, ...],
+        kernel_args: [llvmir.Instruction, ...],
+    ) -> tuple[int, list[llvmir.Instruction]]:
+        """Allocates an LLVM array value to store each flattened kernel arg and
+        another LLVM array to store the typeid for each flattened kernel arg.
+        The arrays are the populated with the LLVM value for each arg.
+        """
+        builder = self._builder
+        context = self._cpu_codegen_targetctx
+
+        meminfos = []
+        for arg_num, argtype in enumerate(kernel_argtys):
+            llvm_val = kernel_args[arg_num]
+
+            meminfos += [
+                meminfo
+                for ty, meminfo in context.nrt.get_meminfos(
```

Comment:
What happens if `argtype` is something that does not have an associated meminfo?